### ISSUE TITLE
The width of some unicode characters is wrong

### ISSUE THREAD

**Issue by dveeden** [dbcli/mycli#149]
Data: https://github.com/dveeden/mysqlunicodedata

In general mycli is better in figuring out the correct width of a column, but not always.

```
mysql msandbox@localhost:unicodedata> SELECT `char`, name FROM ucd WHERE MATCH(name) AGAINST('+COMBINING +CYRILLIC' IN BOOLEAN MODE) ORDER BY NAME LIMI
                                      T 5;
+--------+-------------------------------------------+
| char   | name                                      |
|--------+-------------------------------------------|
| ҅       | COMBINING CYRILLIC DASIA PNEUMATA         |
| ꙱      | COMBINING CYRILLIC HUNDRED MILLIONS SIGN  |
| ҈      | COMBINING CYRILLIC HUNDRED THOUSANDS SIGN |
| ꙼       | COMBINING CYRILLIC KAVYKA                 |
| ⷶ       | COMBINING CYRILLIC LETTER A               |
+--------+-------------------------------------------+
5 rows in set
Command Time: 0.000s
Format Time: 0.001s
```

**Comment by amjith** [comment]
This seems like an issue with the upstream library [wcwidth](https://github.com/jquast/wcwidth). This library is used to determine the display length of a string with unicode chars. 

I'd highly encourage filing an issue with that library.

**Comment by amjith** [comment]
@dveeden Thank you for reporting it to wcwidth. I appreciate you taking the time to test it. :)

**Comment by jquast** [comment]
Suggest close: This was resolved with the release of wcwidth 0.1.5 https://github.com/jquast/wcwidth/releases/tag/0.1.5

This can resolved in your environment by issuing command ``pip install --upgrade`` of mycli, its dependency python-prompt-toolkit, or directly wcwidth. Thanks for your time filing detailed report @dveeden !

**Comment by tsroten** [comment]
@dveeden Thanks for submitting the bug report to wcwidth. And @jquast, thanks for fixing it! Things look resolved, so I'm marking this as closed.
